sdk/monitor/monitor-opentelemetry-exporter/samples/basic-tracer-node/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "example-basic-tracer-node",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Example of using @opentelemetry/tracing in Node.js",
+  "main": "index.js",
+  "scripts": {
+    "build": "echo 'nothing to build'",
+    "test": "echo 'nothing to test'",
+    "lint": "echo 'nothing to lint'",
+    "start": "cross-env APPINSIGHTS_INSTRUMENTATIONKEY=ikey node --inspect ./index.js"
```

Comment:
should this use dotenv instead of cross-env to set the key?

Reply:
Agreed. Switched to add dotenv to all of these samples